scripts/utils/bdlClassnameManager

```diff
@@ -33,8 +33,18 @@ const conversionMap = {
     // 'btn-primary': 'bdl-Button--primary',
     // 'is-disabled': 'bdl-is-disabled',
     // 'label': 'bdl-Label',
-    // 'pill': 'bdl-Pill',
-    // 'pill-error': 'bdl-Pill--error',
+    'pill': 'bdl-Pill',
+    'pill-cloud-button': 'bdl-PillCloud-button',
+    'pill-cloud-container': 'bdl-PillCloud',
+    'pill-error': 'bdl-Pill--error',
+    'pill-selector-hidden-input': 'bdl-PillSelector-input--hidden',
+    'pill-selector-input': 'bdl-PillSelector-input',
+    'pill-selector-input-wrapper': 'bdl-PillSelector',
+    'pill-selector-suggestions-enabled': 'bdl-PillSelector--suggestionsEnabled',
+    'pill-selector-wrapper': 'bdl-PillSelectorWrapper',
```

Comment:
I think we typically try to avoid wrapper/container terminology -- would `bdl-PillSelectorDropdown` work in this case?

Reply:
Good point. I'll look into changing this class name